NLog.Windows.Forms.Tests/NLog.Windows.Forms.Tests.csproj

```diff
@@ -26,7 +26,10 @@
   <ItemGroup>
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.extensibility.execution" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
```

Comment:
is there a reason to update this one? Or newer=better?

Reply:
newer=better :)